src/Features/LanguageServer/Protocol/Handler/OnAutoInsert/OnAutoInsertHandler.cs

```diff
@@ -57,16 +60,14 @@ internal class OnAutoInsertHandler : AbstractStatelessRequestHandler<LSP.Documen
             var service = document.GetRequiredLanguageService<IDocumentationCommentSnippetService>();
 
             // We should use the options passed in by LSP instead of the document's options.
-            var options = await document.GetOptionsAsync(cancellationToken).ConfigureAwait(false);
-            var updatedOptions = options
-                .WithChangedOption(FormattingOptions.UseTabs, !autoInsertParams.Options.InsertSpaces)
-                .WithChangedOption(FormattingOptions.TabSize, autoInsertParams.Options.TabSize);
+            var documentOptions = await ProtocolConversions.FormattingOptionsToDocumentOptionsAsync(
+                request.Options, document, cancellationToken).ConfigureAwait(false);
 
             // The editor calls this handler for C# and VB comment characters, but we only need to process the one for the language that matches the document
-            if (autoInsertParams.Character == "\n" || autoInsertParams.Character == service.DocumentationCommentCharacter)
+            if (request.Character == "\n" || request.Character == service.DocumentationCommentCharacter)
```

Comment:
For consistency, I changed this from `autoInsertParams` -> `request` since that's the variable name all other handlers use.